demo-apps/cf-plugtest-server/src/main/java/org/eclipse/californium/plugtests/PlugtestServer.java

```diff
@@ -523,6 +540,40 @@ public static boolean console(ActiveInputReader reader, long timeout) {
 		return false;
 	}
 
+	/**
+	 * Initializes an OSCORE context for the server built on a pre-defined
+	 * configuration and adds it to the OSCORE context database. The created
+	 * context will support the Appendix B.2. context rederivation procedure.
+	 * 
+	 * @param db the OSCORE context database
+	 */
+	public static void initOscore(HashMapCtxDB db) {
+		AlgorithmID alg = AlgorithmID.AES_CCM_16_64_128;
+		AlgorithmID kdf = AlgorithmID.HKDF_HMAC_SHA_256;
+
+		byte[] master_secret = { 0x01, 0x02, 0x03, 0x04, 0x05, 0x06, 0x07, 0x08, 0x09, 0x0A, 0x0B, 0x0C, 0x0D, 0x0E,
+				0x0F, 0x10 };
+		byte[] master_salt = { (byte) 0x9e, (byte) 0x7c, (byte) 0xa9, (byte) 0x22, (byte) 0x23, (byte) 0x78,
+				(byte) 0x63, (byte) 0x40 };
+		byte[] sid = new byte[] { 0x02 };
+		byte[] rid = new byte[] { 0x01 };
+		serverRid = rid;
```

Comment:
That may be returned instead of written to a `static` field.

Reply:
Done in commit 95a165891b5fec3c4bb8eaacdc1e14da386bd6b8